Cargo.toml

```diff
@@ -1,17 +1,20 @@
 [package]
 name = "trin"
 version = "0.1.0"
-authors = ["Jacob Kaufmann <jacobkaufmann18@gmail.com>", "Jason Carver <ut96caarrs@snkmail.com>"]
+authors = ["https://github.com/ethereum/trin/graphs/contributors"]
 edition = "2021"
 rust-version = "1.66.1"
 default-run = "trin"
 repository = "https://github.com/ethereum/trin"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+license = "GPL-3.0"
```

Comment:
I don't think we use GPL anywhere else. Are we using libraries that force us to use GPL (because they are GPL too)? Most everything in snake charmers uses the MIT license. I know Piper feels strongly about not using GPL. We can make our own decision, of course, but we should decide it explicitly all together.